app/controllers/application_controller.rb

```diff
@@ -14,13 +14,16 @@ def load_cart
     @cart = Cart.new(session[:cart_items])
   end
 
-  # def current_user
-  #   @current_user ||= User.find(session[:user_id]) if session[:user_id]
-  # end
-  #
-  # def current_admin?   #? at the end b/c I want it to return true or false, while for current_user, we want to return an object
-  #   current_user && current_user.admin?   #make sure there's a current_user AND check if they are an admin
-  # end
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+  end
+
+  def current_admin?   #? at the end b/c I want it to return true or false, while for current_user, we want to return an object
+    current_user && current_user.admin?   #make sure there's a current_user AND check if they are an admin
```

Comment:
Line is too long. [106/80]<br>Unnecessary spacing detected.<br>Missing space after #.